### PR TITLE
web showcase

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

File: src/components/token-family/CollectibleTokenFamily.js [hunk 1]
```diff
@@ -1,22 +1,24 @@
-import PropTypes from 'prop-types';
 import React, { useCallback } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { setOpenFamilyTabs } from '../../redux/openStateSettings';
 import { UniqueTokenRow } from '../unique-token';
 import TokenFamilyWrap from './TokenFamilyWrap';
 
 const CollectibleTokenFamily = ({
+  external,
```
Comment (jinchung) [inline src/components/token-family/CollectibleTokenFamily.js:8]: alphabetize

File: src/components/token-family/TokenFamilyHeader.js [hunk 2]
```diff
@@ -68,6 +68,7 @@ const TokenFamilyHeader = ({
   onPress,
   testID,
   title,
+  forceOpen,
```
Comment (jinchung) [inline src/components/token-family/TokenFamilyHeader.js:71]: alphabetize

File: src/handlers/deeplinks.js [hunk 3]
```diff
@@ -19,8 +22,16 @@ export default function handleDeeplink(url) {
         handleWalletConnect(uri);
         break;
       }
-      default:
-        logger.log('unknown deeplink');
+      default: {
+        const addressOrENS = urlObj.pathname.split('/')[1];
```
Comment (jinchung) [inline src/handlers/deeplinks.js:26]: do we want some optional chaining here for the split and index?

File: src/handlers/deeplinks.js [hunk 4]
```diff
@@ -19,8 +22,16 @@ export default function handleDeeplink(url) {
...
+        if (checkIsValidAddressOrDomain(addressOrENS)) {
+          return Navigation.handleAction(Routes.SHOWCASE_SHEET, {
+            address: addressOrENS,
+          });
+        } else {
+          Alert.alert('Invalid deeplink');
```
Comment (jinchung) [inline src/handlers/deeplinks.js:32]: @christianbaroni do we want a different copy here

File: src/hooks/useAccountProfile.js [hunk 5]
```diff
@@ -9,11 +9,20 @@ import useWallets from './useWallets';
 export default function useAccountProfile() {
   const wallets = useWallets();
   const { selectedWallet, walletNames } = wallets;
-
-  const { network } = useAccountSettings();
-  const settings = useAccountSettings();
-  const { accountAddress } = settings;
-
+  const { network, accountAddress } = useAccountSettings();
```
Comment (jinchung) [inline src/hooks/useAccountProfile.js:12]: alphabetize

File: src/hooks/useAccountProfile.js [hunk 6]
```diff
@@ -9,11 +9,20 @@ import useWallets from './useWallets';
...
+  return getAccountProfileInfo(
+    selectedWallet,
+    walletNames,
+    network,
+    accountAddress
+  );
+}
+export function getAccountProfileInfo(
```
Comment (jinchung) [inline src/hooks/useAccountProfile.js:20]: insert line break above

File: src/screens/ImportSeedPhraseSheet.js [hunk 7]
```diff
@@ -142,13 +142,10 @@ const Sheet = styled(Column).attrs({
   z-index: 1;
 `;
 
-export default function ImportSeedPhraseSheet() {
+export function useImportingWallet() {
```
Comment (jinchung) [inline src/screens/ImportSeedPhraseSheet.js:145]: would be nice to keep hooks in hooks/ until we have consensus on a nice new dir structure

File: src/handlers/deeplinks.js [hunk 8]
```diff
@@ -19,8 +22,16 @@ export default function handleDeeplink(url) {
...
-      default:
-        logger.log('unknown deeplink');
+      default: {
+        const addressOrENS = urlObj.pathname?.split('/')?.[1] || '';
+        if (checkIsValidAddressOrDomain(addressOrENS)) {
+          return Navigation.handleAction(Routes.SHOWCASE_SHEET, {
+            address: addressOrENS,
+          });
+        } else {
+          Alert.alert('Invalid deeplink');
```
Comment (jinchung) [inline src/handlers/deeplinks.js:32]: New copy: "Uh oh! We couldn’t recognize this URL!”

File: src/redux/uniqueTokens.js [hunk 9]
```diff
@@ -156,25 +182,57 @@ export default (state = INITIAL_UNIQUE_TOKENS_STATE, action) => {
...
       };
     case UNIQUE_TOKENS_GET_UNIQUE_TOKENS_REQUEST:
-      return {
-        ...state,
-        fetchingUniqueTokens: true,
-      };
+      if (action.showcase) {
+        return {
+          ...state,
+          fetchingUniqueTokens: true,
```
Comment (jinchung) [inline src/redux/uniqueTokens.js:188]: just want to double check this flag isn't flipped (like should `fetchingUniqueTokensShowcase` be here, and should `fetchingUniqueTokens` be on line 193?)

File: src/screens/ImportSeedPhraseSheet.js (hunk 7 above)
Comment (jinchung) [reply src/screens/ImportSeedPhraseSheet.js:145]: can do this after

**Comment by osdnk** [comment]
@jinchung @skylarbarrera 
I think it's ready to do. May I ask for approval?

**Review by jinchung — APPROVED** [review]
🐱

**Comment by brunobar79** [comment]
@skylarbarrera  merging this. Feel free to test on Testflight and I can fix separately

**Pull request merged**